assisted_compliance.md

```diff
@@ -0,0 +1,45 @@
+## Title
+Helping repo owners be compliant by writing their CONTRIBUTING.md for them as a pull request
+
+### Alternate titles
+* Let us help you be compliant
+* Assisted compliance
+
+## Problem
+The team that owns the repository doesn't have a CONTRIBUTING.md; the task force needs them to have this to submit bug fixes.
+
+## Context
+* Teams owning the repository resist compliance-related mandates for having a CONTRIBUTING.md
```

Comment:
Would it be more accurate to say that they are resisting or just that they aren't getting around to it because they continually have other things that are taking their time (as implied by the 3rd bullet point)?

Reply:
Maybe we could say: Teams owning the repository are not complying with mandates for having a CONTRIBUTING.md

Reply:
That works - a simple statement of fact without connotation of intent.

Reply:
The modified sentence is in the pattern now. Resolving.
> Teams owning the repository are not complying with mandates for having a CONTRIBUTING.md